src/Faker/Provider/ro_RO/Text.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Faker\Provider\ro_RO;
+
+class Text extends \Faker\Provider\Text
+{
+    /**
+     * Title: Neghiniţă
+     *
+     * Author: Barbu Ştefănescu Delavrancea
+     *
+     * Language: Romanian
+     *
+     * @see http://www.povesti-pentru-copii.com/barbu-stefanescu-delavrancea/neghinita.html
```

Comment:
The site mentions a copyright, and we need a public license text. Also, please make a separate PR for that.